src/mlx/warnings.py

```diff
@@ -173,8 +173,13 @@ def config_parser_json(self, config):
             try:
                 if bool(config[checker.name]['enabled']):
                     self.activate_checker(checker)
-                    self.get_checker(checker.name).set_maximum(int(config[checker.name]['max']))
-                    self.get_checker(checker.name).set_minimum(int(config[checker.name]['min']))
+                    checker.set_maximum(int(config[checker.name]['max']))
+                    checker.set_minimum(int(config[checker.name]['min']))
```

Comment:
I would say self.get_checker functtion is there to protect agains i think unexsmisting checkers? I would keep it that way...

Reply:
The `checker` variable is an element from the `WarningsPlugin.public_checkers` list. It has been "activated" in a line above like so: `self.activate_checker(checker)`. Using the `self.get_checker` function merely tests the functionality of the `self.activate_checker` function in this use case.

Reply:
yes, but that is the case. Because at this moment in time we use activated checker and with current change we blindly just use any checker as input parameter. Strangely the unit tests did not fail...

Reply:
This code is only reached when the value for key `"enabled"` is `True`. The elements in `public_checkers` list are initialized class objects that get added to `activated_checkers` dict by `self.activate_checker(checker)`.  No new WarningsChecker objects get instantiated here.

Reply:
True. With current flow the checker needs to be enabled before this line is executed anyway.